libsolidity/parsing/DocStringParser.cpp

```diff
@@ -16,16 +16,39 @@ static inline string::const_iterator skipLineOrEOS(
 	return (_nlPos == _end) ? _end : ++_nlPos;
 }
 
-static inline string::const_iterator firstSpaceOrNl(
+static inline string::const_iterator firstSpaceOrTab(
 	string::const_iterator _pos,
 	string::const_iterator _end
 )
 {
 	auto spacePos = find(_pos, _end, ' ');
-	auto nlPos = find(_pos, _end, '\n');
-	return (spacePos < nlPos) ? spacePos : nlPos;
+	auto tabPos = find(_pos, _end, '\t');
+	return (spacePos < tabPos) ? spacePos : tabPos;
+}
+
+static inline string::const_iterator firstWsOrNl(
```

Comment:
Since there's `skipWhitespace` below I would suggest to go with `firstWhitespaceOrNl` here? Would be nice to get rid of `Nl` too :)